app/src/main/java/com/xmartlabs/template/ApplicationComponent.java

```diff
@@ -1,15 +1,14 @@
 package com.xmartlabs.template;
 
-import com.xmartlabs.base.core.CoreComponent;
 import com.xmartlabs.base.core.controller.Controller;
+import com.xmartlabs.base.core.controller.SharedPreferencesController;
 import com.xmartlabs.base.core.module.CoreAndroidModule;
 import com.xmartlabs.base.core.module.GsonModule;
 import com.xmartlabs.base.core.module.LoggerModule;
 import com.xmartlabs.base.core.module.OkHttpModule;
 import com.xmartlabs.base.core.module.PicassoModule;
-import com.xmartlabs.base.core.module.RestServiceModule;
+import com.xmartlabs.base.retrofit.module.RestServiceModule;
```

Comment:
Shouldn't be `com.xmartlabs.base.service.` instead of `com.xmartlabs.base.retrofit.`